### DISCUSSION TITLE
CellAmbasador not found in MOSAIC 20.0 release

### DISCUSSION THREAD

**Discussion by krzycho2** [eclipse/mosaic#46] [Q&A] [answered]
Hello,
In Barnim Basic tutorial It is said "In order to enable communication via cellular networks in this scenario, you need to enable the cell simulator by setting the field active to true". So I enabled cell communication. But then I got critical error "ClassNotFoundException: org.eclipse.mosaic.fed.cell.ambassador.CellAmbassador". What am I missing?
I'm using Eclipse Mosaic 20.0 on Ubuntu 20.04. Version of JRE is 1.8.0_282 so it is Java 8.

**Comment by kschrab** [comment, marked as answer]
Hi, apparently we missed to add the cell-ambassador to our release assembly. I fixed this on our main branch. You have two possibilities to fix this on your side:

* Download [this jar file](https://repo.eclipse.org/content/repositories/mosaic/org/eclipse/mosaic/mosaic-cell/20.0/mosaic-cell-20.0.jar) and put it into the `lib/mosaic` directory.
* Remove your installation of MOSAIC and use [this release bundle](https://ci.eclipse.org/mosaic/job/mosaic/job/main/lastSuccessfulBuild/artifact/bundle/target/eclipse-mosaic-20.1-SNAPSHOT.zip) which is based on the latest main branch instead.

**Reply by krzycho2** [reply]
Thank you. After replacing MOSAIC 20.0 with MOSAIC 20.1, cell communication works fine.
